website/src/utils/yamlDataUtils/delete.js

```diff
@@ -5,6 +5,8 @@ export const deleteData = (path, delData, yamData) => {
     if(path.includes('connections')){
         if(confirm('Deleting this connection will delete the enitites attached to this connection.') === true){
             data = deleteConnData(delData, data)
+        } else {
```

Comment:
Please ask and confirm the user before doing any deletion. 
DONT undo on user cancel. 
just dont perform the action without the user permissions,